powerauth-push-server/src/main/java/io/getlime/push/service/AppRelatedPushClientCacheLoader.java

```diff
@@ -86,8 +90,12 @@ private AppRelatedPushClient createPushClient(final AppCredentialsEntity credent
         pushClient.setAppCredentials(credentials);
 
         if (credentials.getApnsPrivateKey() != null) {
-            final ApnsClient apnsClient = pushSendingWorker.prepareApnsClient(credentials);
-            pushClient.setApnsClient(apnsClient);
+            final ApnsClient apnsClientProduction = pushSendingWorker.prepareApnsClient(credentials, ApnsEnvironment.PRODUCTION);
+            pushClient.setApnsClientProduction(apnsClientProduction);
+            if (configuration.isApnsUseDevelopment()) {
```

Comment:
This is actually on the credentials level too. so there should be something like:
` if (configuration.isApnsUseDevelopment() || credentials.getIosEnvironment()) {`

Reply:
Added.